src/structures/GuildChannel.js

```diff
@@ -36,8 +36,8 @@ class GuildChannel extends Channel {
      */
     this.guild = guild;
 
-    this.parentID = null;
-    this.permissionOverwrites = new Collection();
+    this.parentID = this.parentID || null;
+    this.permissionOverwrites = this.permissionOverwrites || new Collection();
```

Comment:
```suggestion
    this.parentID ??= null;
    this.permissionOverwrites ??= new Collection();
```

Reply:
I'd also originally written `||=` when I made this change locally and eslint got angry at me -- Node version bump when:tm:?

Reply:
We'll probably bump Node.js at the end of the year (around December) for Node.js v16 to become LTS and give users a grace period to upgrade if they didn't by that time.